v2/tutorial/go.mod

```diff
@@ -6,6 +6,8 @@ require (
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/shirou/gopsutil v2.20.9+incompatible
-	github.com/solarwinds/snap-plugin-lib/v2 v2.0.0-beta-2
+	github.com/solarwinds/snap-plugin-lib/v2 v2.0.4
```

Comment:
why is this line here given the local replace rule?

Reply:
it's for better development experience - when you are chaning lib you don't have to tag it or temporary changing branch rule in go.mod. 
Replace is ignored when module is imported directly.
Observed that this pattern is used in other repositories.